poc/field.sage

```diff
@@ -102,6 +102,19 @@ class Field2(Field):
     # Operational parameters
     gf = GF(MODULUS)
 
+    # Return `inp` unmodified if `self == 1`; otherwise return the
+    # all-zero string of the same length.
+    #
+    # Implementation note: To protect the code from timing side channels,
+    # it is important to implement this algorithm in constant time.
+    def mask(self, inp: Bytes) -> Bytes:
```

Comment:
This is sometimes called ["conditional select"](https://docs.rs/subtle/latest/subtle/trait.ConditionallySelectable.html). I'm not sure if there is a widely used term for this.

Reply:
I think `conditional_select` is more descriptive.

Reply:
Done.